### PR TITLE
Replace inline code by snippets

### PR DESCRIPTION
**Pull request by riccikl**
This addresses #364 and replaces some of the remaining occurrences of inline code by code snippets. There is still a lot of inline code left. If these changes are accepted, I might replace some more later.

### CODE REVIEW

**Comment by riccikl** [comment]
Added the remaining changes and rebased onto master

File: entity-framework/core/modeling/relational/sequences.md [hunk 1]
```diff
@@ -24,67 +24,12 @@ You can not configure a sequence using Data Annotations.
...
-}
-
-public class Order
-{
-    public int OrderId { get; set; }
-    public int OrderNo { get; set; }
-    public string Url { get; set; }
-}
-```
+[!code-csharp[Main](../../../../samples/core/Modeling/FluentAPI/Relational/SequenceUsed.cs?name=default&highlight=11,12,13)]
```
Comment (AndriySvyryd) [inline entity-framework/core/modeling/relational/sequences.md]: Only line 13 needs to be highlighted

File: entity-framework/core/modeling/shadow-properties.md [hunk 2]
```diff
@@ -66,22 +42,4 @@ You can use the Fluent API to configure shadow properties. Once you have called
...
-    }
-}
-
-public class Blog
-{
-    public int BlogId { get; set; }
-    public string Url { get; set; }
-}
-```
+[!code-csharp[Main](../../../samples/core/Modeling/FluentAPI/ShadowProperty.cs?name=ShadowProperty&highlight=7,8)]
```
Comment (AndriySvyryd) [inline entity-framework/core/modeling/shadow-properties.md]: Only line 8 needs highlighting.

File: samples/core/Modeling/Conventions/Relationships/OneToOne.cs [hunk 3]
```diff
@@ -9,6 +9,7 @@ class MyContext : DbContext
         public DbSet<BlogImage> BlogImages { get; set; }
     }
 
+    #region property
```
Comment (AndriySvyryd) [inline samples/core/Modeling/Conventions/Relationships/OneToOne.cs]: Use PascalCase for all region names

**Comment by AndriySvyryd** [comment]
@riccikl Thanks for your contribution!

**Pull request merged**